blockchain/chain_sync/src/sync.rs

```diff
@@ -1,6 +1,8 @@
 // Copyright 2020 ChainSafe Systems
 // SPDX-License-Identifier: Apache-2.0, MIT
 
+#![allow(unused_variables)]
```

Comment:
remove please, not doing anything anyway

Reply:
updated